pandas/core/generic.py

```diff
@@ -10234,28 +10236,37 @@ def _doc_parms(cls):
 
 Returns
 -------
-%(outname)s : %(name1)s or %(name2)s (if level specified)
+%(outname)s : %(name1)s or %(name2)s
+    If level is specified, then, %(name2)s is returned; otherwise, %(name1)s
+    is returned.
 
 %(see_also)s
 %(examples)s"""
 
 _all_doc = """\
 Return whether all elements are True, potentially over an axis.
 
-Returns True if all elements within a series or along a Dataframe
-axis are non-zero, not-empty or not-False."""
+Returns True unless there at least one element within a series or
+along a Dataframe axis that is False or equivalent (e.g. zero or
+empty)."""
 
 _all_examples = """\
 Examples
 --------
-Series
+**Series**
```

Comment:
Is this right?

Reply:
We've been using those bold headers in other places too. I don't have a preference, but is consistent with other docstrings.

Reply:
I did a quick search and this was the only one I found that wasn't bold.